gotests_test.go

```diff
@@ -638,7 +647,7 @@ func toSnakeCase(s string) string {
 	return string(res)
 }
 
-func ignoreTabs(s string) string {
+func gofmt(s string) string {
```

Comment:
This isn't used anywhere and can be removed. (Also, its new name doesn't match its behavior.)